File: Ejercicios/C++/Ejercicio10.cpp
#include <iostream>
#include <list>
using namespace std;
int main() {
    list<string> fila = {"Victor", "Maria", "Jose"};
    while(!fila.empty()) {
        cout << "Despachando a: " << fila.front() << "\n";
        fila.pop_front();
    }
    cout << "No hay mas clientes en fila.\n";
    return 0;
}

File: Ejercicios/Java/Ejercicio10.java
import java.util.LinkedList;
public class Ejercicio10 {
    public static void main(String[] args) {
        LinkedList<String> fila = new LinkedList<>();
        fila.add("Victor"); fila.add("Maria"); fila.add("Jose");
        while(!fila.isEmpty()) {
            System.out.println("Despachando a: " + fila.removeFirst());
        }
        System.out.println("No hay mas clientes en fila.");
    }
}
